api/python/quilt3/telemetry.py

```diff
@@ -137,6 +145,10 @@ def decorated(*args, **kwargs):
         return decorated
 
 
+ApiTelemetry.reset_session_id()
+ApiTelemetry.set_client_context(None)
```

Comment:
For backward-compatibility with old events, we should make sure this shows up as null in the analytics beacon and not 'None' or something like that.